tests/hw9/hw3_test.py

```diff
@@ -0,0 +1,56 @@
+import tempfile
+
+import pytest
+
+from homework9.hw3 import universal_file_counter
+
+
+@pytest.fixture()
+def tmp_files(file_list, values) -> str:
+    temp_dir = tempfile.gettempdir()
+    for i in file_list:
+        temp_file = f"{temp_dir}/file{i}.txt"
+        with open(temp_file, "w") as fw:
+            fw.write(values[int(i)])
+    return temp_dir
```

Comment:
`tempfile.gettempdir()` возвращает дефолтную временную директорию. Она не обязательно пустая. У меня в ней оказалось больше сотни файлов и в итоге тест падал т.к. считал токены в 3х файлах которые создаёт микстура и во всех остальных (получалось более 4000 токенов :) )

Для таких вещей советую использовать контекстные версии временных папок/файлов, которые автоматически генерятся и удаляются https://docs.python.org/3/library/tempfile.html?highlight=tempfile.mkdtemp#tempfile.TemporaryDirectory

```suggestion
    with tempfile.TemporaryDirectory() as temp_dir:
        for i in file_list:
            temp_file = f"{temp_dir}/file{i}.txt"
            with open(temp_file, "w") as fw:
                fw.write(values[int(i)])
        yield temp_dir
```